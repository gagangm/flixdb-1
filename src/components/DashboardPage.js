import React from 'react';
import { connect } from 'react-redux';
import Header from './Header';
import MoviesList from './MoviesList';
import imdb_top_250 from '../database/imdb-top-250';
import loadMovieDetails from '../database/create-list';

const DashboardPage = () => (
    <div className="container">
      <Header />
      <div className="page-container">
        <MoviesList/>
      </div>
    </div>
);

export default DashboardPage;