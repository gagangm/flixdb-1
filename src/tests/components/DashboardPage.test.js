import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('Should render DashboardPage component', () => {
  const wrapper = shallow(<DashboardPage />)

  expect(wrapper).toMatchSnapshot();
});
