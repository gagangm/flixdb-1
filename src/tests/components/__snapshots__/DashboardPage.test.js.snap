// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render DashboardPage component 1`] = `
<div>
  Dashboard Page Content
</div>
`;
